newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview/index.js

```diff
@@ -104,20 +122,26 @@ export default class AnimationPreview extends Component<Props, State> {
           resourcesLoader={resourcesLoader}
           project={project}
         />
-        <Line>
-          <Column expand>
-            <FlatButton
-              icon={<Replay />}
-              label="Replay"
-              onClick={this.replay}
-            />
-            <FlatButton
-              icon={paused ? <PlayArrow /> : <Pause />}
-              label={paused ? 'Play' : 'Pause'}
-              onClick={paused ? this.play : this.pause}
-            />
-          </Column>
-        </Line>
+        <div style={styles.container}>
```

Comment:
What about using a `<Line>` instead of this `div`? Would allow to remove the style and should look correct.
You should also be able to use `<Column>` that will add left and right margins. And reuse Line again but with `noMargin`:

```
<Line>
  <Column>
    <Line noMargin>
      <Timer style={styles.timeIcon} />
      <TextField ... />
    </Line>
  </Column>
  <Column>
    <Line noMargin>
      <FlatButton ../>
      <FlatButton .. />
    </Line>
  </Column>
</Line>
```

(might have a simpler solution, but that should work)

Reply:
The idea is to keep the still to a minimum. Using this, I was able to tweak the margins of ALL the app using a single constant. Having the style out of the rendering also allow later to ease port to potential other platforms (react-native or something else)

Reply:
all done now :)